apis/python/src/tiledbsoma/stats.py

```diff
@@ -11,12 +11,12 @@
 
 
 def tiledbsoma_stats_json() -> str:
-    """Returns tiledbsoma stats as a JSON string"""
+    """Returns tiledbsoma stats as a JSON string."""
```

Comment:
Asymmetry with respect to other methods. Noticed while I was reading Sphinx summaries.